xee/ext.py

```diff
@@ -1037,7 +1052,14 @@ def open_dataset(
       executor_kwargs (optional): A dictionary of keyword arguments to pass to
         the ThreadPoolExecutor that handles the parallel computation of pixels
         i.e. {'max_workers': 2}.
-
+      getitem_kwargs (optional): Exponential backoff kwargs passed into
```

Comment:
Could you update the following arg documentation comments:
        - `initial_delay`, the initial delay in milliseconds before the first retry. Defaults to 500.
        - `max_retries`, the maximum number of retry attempts. Defaults to 6.

No need to mention `ee.data.computePixels`. That's an implementation detail.